apiary.apib

```diff
@@ -1043,6 +1043,97 @@ API endpoints that Operation Code's Rails backend makes available to its React f
             errors: "Some error message"
         }
 
+## Social User | Create [/api/v1/social_users{email,first_name,last_name,zip,password}]
```

Comment:
There needs to be a `?` after the `{` for apiary to recognize all the param individually.  Here is an example:  https://github.com/OperationCode/operationcode_backend/blob/master/apiary.apib#L43

Reply:
done!